releases/nightly-build/compose-files/docker-compose-nexus-mongo-arm64.yml

```diff
@@ -24,6 +24,10 @@ x-common-env-variables: &common-variables
   Clients_CoreData_Host: edgex-core-data
   Clients_Logging_Host: edgex-support-logging
   Logging_EnableRemote: "true"
+  Databases_Primary_Type: mongodb
+  Databases_Primary_Host: edgex-mongo
+  Databases_Primary_Port: 27017
+  support-logging_Writable_Persistence: database
```

Comment:
Have you verified this `support-logging_` prefix works? Also, since this really is `not` a global setting, I would move it to config-seed for now. Once we have self-seeding, it will get moved to support-notifications.

Reply:
This PR says it should https://github.com/edgexfoundry/edgex-go/pull/1461  -- tested?

Reply:
```
environment:
  <<: *common-variables
```

needs to be added to affected microservices for this environment block to be effective.

Reply:
Modified